captcha/events.py

```diff
@@ -53,10 +60,129 @@ async def cleaner(self, member: Member):
         finally:
             await self.delete_challenge_for(member)
 
+    async def skip_challenge(self, author: Member, challenge: Challenge):
+        roles = [
+            challenge.guild.get_role(role)
+            for role in await self.data.guild(challenge.guild).autoroles()
+        ]
+        try:
+            await self.congratulation(challenge, roles)
+            await self.remove_temprole(challenge)
+
+            def ok_check(msg: str):
+                return f"✅ {msg}"
+
+            await self.send_or_update_log_message(
+                challenge.guild,
+                ok_check(f"Captcha skiped by {author.name} ({author.id})."),
```

Comment:
```suggestion
            await self.send_or_update_log_message(
                challenge.guild,
                f"✅ Captcha skiped by {author.name} ({author.id}).",
```
We can just add it into the string instead of creating a function.